runtime/caml/mlvalues.h

```diff
@@ -84,13 +84,68 @@ typedef opcode_t * code_t;
 #define Unsigned_long_val(x) ((uintnat)(x) >> 1)
 #define Unsigned_int_val(x)  ((int) Unsigned_long_val(x))
 
-/* Encoded exceptional return values, when functions are suffixed with
-   _exn. Encoded exceptions are invalid values and must not be seen
-   by the garbage collector. */
+/* A 'result' type/monad for OCaml computations. */
```

Comment:
The "M" word is known to inspire terror in many programmers, cf. SPJ's "warm fuzzy things" alternate naming.  Maybe just "A 'result' type" ?